BlueprintUI/Sources/Layout/Column.swift

```diff
@@ -43,13 +43,48 @@ public struct Column: StackElement {
 
     public var children: [(element: Element, traits: StackLayout.Traits, key: AnyHashable?)] = []
 
-    public private(set) var layout = StackLayout(
-        axis: .vertical,
-        alignment: ColumnAlignment.leading.stackAlignment
-    )
+    public private(set) var layout: StackLayout = .defaultColumn
 
     public init() {}
 
+    /// Initializer using result builder to declaritively build up a stack.
+    /// - Parameters:
+    ///   - layout: A StackLayout describing the column's layout
+    ///   - elements: A block containing all elements to be included in the stack.
+    /// - Note: If element is a StackChild, then traits and key will be pulled from the element, otherwise defaults are passed through.
+    public init(
+        layout: StackLayout = .defaultColumn,
+        @ElementBuilder<StackChild> elements: () -> [StackChild]
+    ) {
+        self.init(elementsBuilder: elements)
+        self.layout = layout
+    }
+
+    /// Initializer using result builder to declaritively build up a stack.
+    /// - Parameters:
+    ///   - alignment: Determines the cross-axis layout
+    ///   - verticalUnderflow: Determines the layout when there is extra free space available.
+    ///   - verticalOverflow: Determines the layout when there is not enough space to fit all children as measured.
+    ///   - horizontalAlignment: Specifies how children will be aligned horizontally.
+    ///   - minimumVerticalSpacing: Spacing in between elements.
+    ///   - elements: A block containing all elements to be included in the stack.
+    /// - Note: If element is a StackChild, then traits and key will be pulled from the element, otherwise defaults are passed through.
+    public init(
+        alignment: StackLayout.Alignment = StackLayout.defaultColumn.alignment,
+        verticalUnderflow: StackLayout.UnderflowDistribution = StackLayout.defaultColumn.underflow,
```

Comment:
API design question: I wonder if we should remove "vertical" and "horizontal" from these; since that's implicit from being a Column.